### ISSUE TITLE
Changing webserver port, mysql and rabbitmq

### ISSUE THREAD

**Issue by mitmelon** [mattrayner/docker-lamp#132]
Please how do I change this ports because its conflicting with existing mysql on docker

**Comment by pzhlkj6612** [comment]
Hello. Are you going to export the 3306 port from the container to your host? Please see [Publish or expose port (-p, --expose) § docker run | Docker Documentation](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose).

**Comment by pzhlkj6612** [comment]
> ... because its conflicting with existing mysql on docker

Well, did you install another mysql in the container?

**Comment by mitmelon** [comment]
Yes, Actually I installed postal mail service which uses mysql too.
